packages/html/src/tabstrip/tests/tabstrip-scrollable-buttons-hidden.tsx

```diff
@@ -0,0 +1,163 @@
+import { TabStripContent, TabStripItem, TabStripLeft, TabStripNormal } from '..';
+
+
+const style = ``;
+
+export default () => (
+    <>
+        <style>{style}</style>
+        <div id="test-area" className="k-d-grid k-grid-cols-3">
+
+            <span>Horizontal orientation</span>
+            <span></span>
+            <span></span>
+             <section>
+                <TabStripNormal scrollable scrollButtonsPosition="hidden" scrollingPosition="start"
```

Comment:
Hidding the scroll buttons through `scrollButtonsPosition` sound really weird to me, could we change that?

Reply:
As discussed offline, the prop name should be changed to **scrollButtons**